tests/Integration/inc/Engine/Optimization/DynamicLists/Subscriber/addPreloadExcluisons.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+namespace WP_Rocket\Tests\Integration\inc\Engine\Optimization\DynamicLists\Subscriber;
+
+use WP_Rocket\Tests\Integration\TestCase;
+
+/**
+ * @covers \WP_Rocket\Engine\Optimization\DynamicLists\Subscriber::add_preload_exclusions()
+ *
+ * @group  DynamicLists
+ */
+class Test_AddPreloadExcluisons extends TestCase {
```

Comment:
```suggestion
class Test_AddPreloadExclusions extends TestCase {
```

+ same typo in the filename